spec/installer-spec.coffee

```diff
@@ -1,38 +1,29 @@
 fs = require 'fs'
 path = require 'path'
-
-grunt = require 'grunt'
+jetpack = require 'fs-jetpack'
 temp = require 'temp'
+{build} = require '../index'
 
 describe 'create-windows-installer task', ->
-  beforeEach ->
-    updateExePath = path.join(__dirname, 'fixtures', 'app', 'Update.exe')
-    fs.unlinkSync(updateExePath) if fs.existsSync(updateExePath)
-
-  it 'creates a nuget package and installer', ->
-    outputDirectory = temp.mkdirSync('grunt-electron-installer-')
 
-    grunt.config.init
-      pkg: grunt.file.readJSON(path.join(__dirname, 'fixtures', 'app', 'resources', 'app', 'package.json'))
+  appDirectory = jetpack.cwd(__dirname, 'fixtures', 'app')
 
-      'create-windows-installer':
-        config:
-          appDirectory: path.join(__dirname, 'fixtures', 'app')
-          outputDirectory: outputDirectory
+  beforeEach ->
+    updateExePath = appDirectory.path('Update.exe')
+    jetpack.remove(updateExePath) if jetpack.exists(updateExePath)
 
-    grunt.loadTasks(path.resolve(__dirname, '..', 'tasks'))
+  it 'creates a nuget package and installer', ->
+    outputDirectory = jetpack.cwd(temp.mkdirSync('electron-installer-windows'))
 
-    tasksDone = false
-    grunt.registerTask 'done', 'done',  -> tasksDone = true
-    grunt.task.run(['create-windows-installer', 'done']).start()
+    config =
+          appDirectory: appDirectory.path()
```

Comment:
This should use a two space indent.

I just pushed a change to master to lint the specs so you should see errors now on this branch if you merge/rebase and then run `grunt lint`.